libsrc/leddevice/LedDeviceLpd8806.cpp

```diff
@@ -21,7 +21,7 @@ bool LedDeviceLpd8806::init(const QJsonObject &deviceConfig)
 	_ledBuffer.resize(messageLength, 0x00);
 
 	// Perform an initial reset to start accepting data on the first led
-	return writeBytes(clearSize, _ledBuffer.data()) >= 0;
+	return writeBytes(_ledBuffer.size(), _ledBuffer.data());
```

Comment:
I'm not sure if we should replace clearsize, because original driver does the same:
https://github.com/hyperion-project/hyperion/blob/master/libsrc/leddevice/LedDeviceLpd8806.cpp#L30